addons/mail/wizard/mail_template_preview_view.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="mail_template_preview_view_form" model="ir.ui.view">
+            <field name="name">mail.template.preview.view.form</field>
+            <field name="model">mail.template.preview</field>
+            <field name="arch" type="xml">
+                <form string="Email Preview">
+                    <h3>Preview of <field name="mail_template_id" readonly="1" nolabel="1" class="oe_inline text-primary" options="{'no_open' : True}"/></h3>
+                    <div class="alert alert-danger" role="alert" attrs="{'invisible' : [('error_msg', '=', False)]}">
+                        <field name="error_msg" />
+                    </div>
+                    <div class="o_row">
+                        <span>Choose an example <field name="model_id" readonly="1"/> record:</span>
+                        <div>
+                            <field name="resource_ref" class="oe_inline" 
+                                options="{'hide_model': True, 'no_create': True, 'no_edit': True, 'no_open': True}"/>
+                        </div>
+                    </div>
+                    <div class="o_row">
+                        <span>Force the language: </span><field name="lang" class="oe_inline pr-2"/>
+                    </div>
+                    <group>
+                        <field name="subject" readonly="1"/>
+                        <field name="email_from" readonly="1"
+                            attrs="{'invisible':[('email_from','=', False)]}"/>
+                        <field name="partner_ids" widget="many2many_tags" readonly="1"
+                            attrs="{'invisible':[('partner_ids', '=', [])]}"/>
+                        <field name="email_to" readonly="1"
+                            attrs="{'invisible':[('email_to','=', False)]}"/>
+                        <field name="email_cc" readonly="1"
+                            attrs="{'invisible':[('email_cc','=', False)]}"/>
+                        <field name="reply_to" readonly="1"
+                            attrs="{'invisible':[('reply_to','=', False)]}"/>
+                        <field name="scheduled_date" readonly="1"
+                            attrs="{'invisible':[('scheduled_date','=', False)]}"/>
```

Comment:
As all those fields are computed, you should not need to specify `readonly="1"` everywhere.